.github/CODEOWNERS

```
@@ -0,0 +1,19 @@
+#
+# codeowners reference:
+#   https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+#
+# for ease of comparison, usernames are kept alphabetized
+#
+
+# default rule
+* @aaschaer @ada-globus @derek-globus @kurtmckee @sirosen
+
+# Flows service
+/src/globus_sdk/services/flows/ @ada-globus @derek-globus @kurtmckee @sirosen
+/src/globus_sdk/_testing/data/flows/ @ada-globus @derek-globus @kurtmckee @sirosen
+/tests/functional/services/flows/ @ada-globus @derek-globus @kurtmckee @sirosen
```

Do you think this pattern would be usable here?

```suggestion
**/flows/ @ada-globus @derek-globus @kurtmckee @sirosen
```

---

I wasn't sure. I guess the risk that a directory gets named `flows` for some other purpose is low, and importantly the impact of incorrect matching is also low.
Why don't we go ahead with changing to the `**` match and we can always re-evaluate if we ever find a need. I'll push the update in a few.